components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -526,6 +526,16 @@ private long getTotalUsers(String domainName) throws CharonException {
             } catch (org.wso2.carbon.user.core.UserStoreException e) {
                 throw new CharonException("Error while getting total user count in domain: " + domainName);
             }
+        } else if (domainName != null) {
```

Comment:
shall we use StringUtils.isNotBlank

Reply:
addressed in 677051fc7cacc938fbe165d002024ef39b230883